var/spack/repos/builtin/packages/py-onnx-runtime/package.py

```diff
@@ -126,4 +127,9 @@ def cmake_args(self):
 
     @run_after("install")
     def install_python(self):
-        PythonPackage.install(self, self.spec, self.prefix)
+        platlib = self.spec["python"].package.platlib
+        with working_dir(self.build_directory):
+            py = which("python")
```

Comment:
This won't work for `python~pythoncmd` where only the `python3` executable exists. Spack already has a `python` function injected into the module, so you can use `python(...)` without using `which(...)` first.